ConfigurationSystem/ConfigTemplate.cfg

```diff
@@ -40,9 +40,11 @@ Agents
     AutoModifyUsers = False
     AutoSuspendUsers = False
     VO = None
+    DryRun = False
```

Comment:
I would set the CS default to True, so when the agent is installed it is not doing anything until it is explicitly enabled.